### PR TITLE
Prevent enterprise test to run on forked repo [API-1462]

### PR DESCRIPTION
**Pull request by nevzatseferoglu**
Ensure nightly jobs are only running on original repository.
Remove unnecessary obsolete explanation and fix typos.

### CODE REVIEW

**Review by utku-caglayan — APPROVED** [review]
Looks good!

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ Number of the clients:
...
+#### Expected behavior
 
 
-#### Actual behaviour
+#### Actual behavior
 
 
-#### Steps to reproduce the behaviour
\ No newline at end of file
+#### Steps to reproduce the behavior
```
Comment (yuce) [inline .github/ISSUE_TEMPLATE.md]: `behaviour` is not wrong, it's written like that in British English.

Comment (nevzatseferoglu) [reply .github/ISSUE_TEMPLATE.md]: My bad, but Google itself uses as `behavior`,
https://support.google.com/analytics/answer/2785577?hl=en#zippy=%2Cin-this-article
I think we should choose the same phrase with them as being a Go client.
Also, overall usage of `behavior` is bigger.
Screen Shot 2022-08-01 at 10 07 02
Wdyt ?

Comment (yuce) [reply .github/ISSUE_TEMPLATE.md]: Google is a US company, so that makes sense for them :) I don't agree with your reasoning, but `behavior` is OK.

**Review by yuce — APPROVED** [review]

**Pull request merged**